bin/install-jetpack.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+# Check if nvm is installed
+[ -z "$NVM_DIR" ] && NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
I considered checking to see if nvm was installed via homebrew. In which case we could add a step that sources nvm.sh from `$ brew --prefix nvm`.  
Homebrew does prints out a [significant caveat](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/nvm.rb#L29) when installing nvm via brew and mentions the same issues on the [package page](https://formulae.brew.sh/formula/nvm). NVM also [warns about installing from homebrew as wel](https://github.com/nvm-sh/nvm#important-notes)l.

in general, if nvm is installed properly, `NVM_DIR` should point to the nvm directory.